style.css

```diff
@@ -60,10 +68,54 @@ li {
   color: #6070ff;
 }
 
-.navbar ul {
+.desktop-nav {
   display: none;
 }
 
+.mobile-menu {
+  position: absolute;
+  height: 750px;
```

Comment:
- [ ] As the previous reviewer mentioned your mobile menu is sticking to the top when scrolling down so it is possible to see the content below. I think you could use `position: fixed` and `height: 100vh` so your mobile menu stays on top of the content and occupies 100% of the `viewport height` :wink:

![image](https://user-images.githubusercontent.com/81522435/145884692-241b5cf3-29e0-4212-b803-6c715867dc65.png)

Reply:
Thanks for the suggestion, effect the required changes